tests/test_ticketmaster.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import os
+
+# Add the 'apis' directory to sys.path so Python can find it
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'apis')))
+
+# import TicketmasterAPI and TicketmasterEvent as normal
+from ticketmaster_api import TicketmasterAPI, TicketmasterEvent # type: ignore
```

Comment:
You can just specify directory with period in front of the module name like below:
from apis.ticketmaster_api import TicketmasterAPI, TicketmasterEvent